Server/Meeting.cs

```diff
@@ -26,19 +26,23 @@ public MeetingServices(String username, String topic, int minParticipants, List<
             this.minParticipants = minParticipants;
             this.locDateOptions = slots;
             this.invitees = invitees;
-            this.closed = false;
+            this.isClosed = false;
         }
 
         public string Topic { get => topic; }
+        public string CoordinatorUsername { get => coordinatorUsername; }
         public int MinParticipants { get => minParticipants; }
         public List<(string, DateTime)> LocDateOptions { get => locDateOptions; }
         public List<(string, DateTime)> Slots { get => locDateOptions; }
-        public Boolean Closed { set => closed = true; }
+        public Boolean GetIsClosed { get => isClosed; }
+        public Boolean IsClosed { set => isClosed = true; }
```

Comment:
Ikke intuitivt at å kalle meeting.isClosed og så blir bool automatisk satt til True. Selv om man har GetIsClosed så er IsClosed ikke et bra navn for en setter.

Reply:
Ok, trodde det var god praksis å ha "is" foran boolske variabler

Reply:
Ved å ha "is" foran skjønner man at det er en boolean bare ved å lese navnet
https://www.samanthaming.com/tidbits/34-better-boolean-variable-names

Reply:
ååå, nå leste jeg koden bedre! skjønner hva du mener

Reply:
Endret navnet ``Closed`` til ``IsClosed`` litt kjapt

Reply:
Men det var opprinnelig `Closed`. Synes du det er bedre? Jeg synes kanskje `SetMeetingToClosed`
 fungerer bedre selvom det blir litt langt

Reply:
Closed var også dårlig :) SetMeetingToClosed er mye bedre! :)